File: src/main/java/com/iflat/xr/bean/SrProjectPrice.java
package com.iflat.xr.bean;

import java.util.Date;

/**
 * 单船结算单价
 */
public class SrProjectPrice {

    private String id;
    private String projNo;
    private String projName;
    private String code;
    private String category;
    private String content;
    private String specs;
    private double degree;
    private String unit;
    private double quota;
    private double price;
    private String attachment;
    private String comment;
    private boolean isQuota;  // 需定额

    private String creatorAcc;
    private String creatorName;
    private Date createTime;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getProjNo() {
        return projNo;
    }

    public void setProjNo(String projNo) {
        this.projNo = projNo;
    }

    public String getProjName() {
        return projName;
    }

    public void setProjName(String projName) {
        this.projName = projName;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getSpecs() {
        return specs;
    }

    public void setSpecs(String specs) {
        this.specs = specs;
    }

    public double getDegree() {
        return degree;
    }

    public void setDegree(double degree) {
        this.degree = degree;
    }

    public String getUnit() {
        return unit;
    }

    public void setUnit(String unit) {
        this.unit = unit;
    }

    public double getQuota() {
        return quota;
    }

    public void setQuota(double quota) {
        this.quota = quota;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public String getAttachment() {
        return attachment;
    }

    public void setAttachment(String attachment) {
        this.attachment = attachment;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

    public boolean getIsQuota() {
        return isQuota;
    }

    public void setIsQuota(boolean isQuota) {
        this.isQuota = isQuota;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }
}

File: web/app/model/xr/SrProjectPrice.js
Ext.define('iFlat.model.xr.SrProjectPrice', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'srProjectPrice.id', mapping: 'id', type: 'string'},
        {name: 'srProjectPrice.projNo', mapping: 'projNo', type: 'string'},
        {name: 'srProjectPrice.projName', mapping: 'projName', type: 'string'},
        {name: 'srProjectPrice.code', mapping: 'code', type: 'string'},
        {name: 'srProjectPrice.category', mapping: 'category', type: 'string'},
        {name: 'srProjectPrice.unit', mapping: 'unit', type: 'string'},
        {name: 'srProjectPrice.price', mapping: 'price', type: 'number'},
        {name: 'srProjectPrice.attachment', mapping: 'attachment', type: 'string'},
        {name: 'srProjectPrice.comment', mapping: 'comment', type: 'string'},
        {name: 'srProjectPrice.isQuota', mapping: 'isQuota', type: 'boolean'},
        {name: 'srProjectPrice.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'srProjectPrice.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'srProjectPrice.createTime', mapping: 'createTime', type: 'date'},
    ]
});